astropy/modeling/functional_models.py

```diff
@@ -33,6 +35,17 @@ class BaseGaussian1D(Fittable1DModel):
     mean = Parameter(default=0)
     stddev = Parameter(default=1)
 
+    def __init__(self, amplitude, mean, stddev, **kwargs):
+        # Ensure stddev makes sense if its bounds are not explicitly set.
+        if 'bounds' not in kwargs:
+            kwargs['bounds'] = {'stddev': (FLOAT_EPSILON, None)}
```

Comment:
I wonder if we should have a more global way of specifying this constant then and have one recommended way for users to say that they want to specify the parameter should be >0